README.md

```diff
@@ -0,0 +1,8 @@
+#Polygon Shape Creator(for Google Maps)
```

Comment:
This Description is for my fork, i believe this does not suite you since you are the creator of the project. so please be sure to update that the way it suits you.